Makefile

```diff
@@ -162,6 +168,12 @@ st-ssl: run-etcd-ssl dist/calicoctl docker calico_test/.calico_test.created busy
 	           calico/test \
 	           sh -c 'cp -ra tests/st/* /tests/st && cd / && nosetests $(ST_TO_RUN) -sv --nologcapture --with-timer $(ST_OPTIONS)'
 
+add-ssl-hostname:
```

Comment:
Might it be a little bad blatting over /etc/hosts with your own host name configuration.